p4-fusion/commands/describe_result.cc

```diff
@@ -13,13 +13,17 @@ void DescribeResult::OutputStat(StrDict* varList)
 int DescribeResult::OutputStatPartial(StrDict* varList)
 {
 	std::string indexString = std::to_string(m_FileData.size());
-	m_FileData.push_back(FileData {});
 
 	StrPtr* depotFile = varList->GetVar(("depotFile" + indexString).c_str());
+	if(depotFile == nullptr){
```

Comment:
```suggestion
	if(!depotFile)
	{
```
We have been using this style throughout the project, probably should also follow it here